### DISCUSSION TITLE
A way to add a MIDI Channel offset ?

### DISCUSSION THREAD

**Discussion by Onosendaifr** [probonopd/MiniDexed#684] [Q&A]
The MIDI controller i use for my Mini Dexed send only data from channel 3. It also send notes to all TG in Omni, and TG 3 when i choose independant. 

I wish i could add a MIDI channel offset to make it send date to channel 9+ instead, without using external gear.

Do you think there is a way to do that ?

**Comment by probonopd** [comment]
Not sure what you are looking for. Why don't you set up TG 1 to listen on MIDI channel 3, TG to listen on MIDI channel 4, and so on?
